processing/python/audience_view/surfaces.py

```diff
@@ -45,12 +45,13 @@ class UtteranceLine:
     cat_surface = None
     
     def __init__(self, s_width, s_height, utt, cat, font, font_bold, pos_x):
+        print("utt {}  cat {}".format(utt,cat))
```

Comment:
A print command in the constructor - always a bad idea

Reply:
done